### ISSUE TITLE
How can I add an color augmentation into the transform.py?

### ISSUE THREAD

**Issue by ZenFSheng** [ZFTurbo/Keras-RetinaNet-for-Open-Images-Challenge-2018#12]
Hi~ I decide to use your model as my pretrained model，but I‘ve found that there's no color augmentation in the generator，and it seems that the keras-retinanet use a modified transform generator， can you help me with that？Thank you very much~

**Comment by ZenFSheng** [comment]
> If you use generator from this repository, it already has some color augmentations on this line:
> 
> https://github.com/ZFTurbo/Keras-RetinaNet-for-Open-Images-Challenge-2018/blob/master/retinanet_training_level_1/open_images_generator.py#L347

Thanks a lot~，I also notice that there are some morphological augmentation in the generator，is it work well for your training ？

**Comment by ZFTurbo** [comment]
Yes. It's the exact augmentations I used previous year. It works fine. Morphological augmentation was used from original RetinaNet repository. This year I rewrote them to use [albumentation ](https://github.com/albu/albumentations)library.

**Comment by ZenFSheng** [comment]
> Yes. It's the exact augmentations I used previous year. It works fine. Morphological augmentation was used from original RetinaNet repository. This year I rewrote them to use [albumentation ](https://github.com/albu/albumentations)library.

Thank you！That’s really helpful to me~
